workflow/automation/lib/MgmtDB.py

```diff
@@ -424,7 +437,7 @@ def get_runnable_tasks(
                 ).fetchall()
                 runnable_tasks.extend(
                     [
-                        (*task, self.get_retries(*task))
+                        (*task, self.get_retries(*task, get_WCT=True))
```

Comment:
So this effectively controls the increase of WCT if it was a WCT failure?

Reply:
Yes as that determines the number of past retries that have hit WCT, so when that is passed into the auto submit scripts then they adjust the WCT based on the number of past retries